RecoMTD/TrackExtender/plugins/TrackExtenderWithMTD.cc

```diff
@@ -321,10 +329,12 @@ namespace {
     if (!addPIDError) {
       //*TODO* deal with heavier nucleons and/or BSM case here?
       float chi2_pi = tofpid.dtchi2;
-      float chi2_k =
-          (tofpid.tmtd - tofpid.dt_k - t_vtx) * (tofpid.tmtd - tofpid.dt_k - t_vtx) / (tofpid.dterror * tofpid.dterror);
-      float chi2_p =
-          (tofpid.tmtd - tofpid.dt_p - t_vtx) * (tofpid.tmtd - tofpid.dt_p - t_vtx) / (tofpid.dterror * tofpid.dterror);
+      float chi2_k = (tofpid.tmtd - tofpid.dt_k - t_vtx) * (tofpid.tmtd - tofpid.dt_k - t_vtx) /
```

Comment:
Again very minor, but you could pre-define `tofpid.dterror * tofpid.dterror - tofpid.sigma_dt_pi * tofpid.sigma_dt_pi`
or `tofpid.dterror2 - tofpid.sigma_dt_pi * tofpid.sigma_dt_pi`, if you take into account my comment above. 
Then you could use this in the denominator of both `chi2_k` and `chi2_p`.  Not sure it actually would improve performance, given compiler optimizations, but it arguably makes the code a bit tidier.

Reply:
Thank you for the comments. I have addressed them in the latest commit 6d4465f